server/models/MemberInvitation.js

```diff
@@ -9,6 +9,35 @@ import sequelize, { DataTypes } from '../lib/sequelize';
 
 export const MEMBER_INVITATION_SUPPORTED_ROLES = [roles.ACCOUNTANT, roles.ADMIN, roles.MEMBER];
 
+const sendInvitationEmail = async (invitation, memberParams, models, sequelizeParams, collective, skipDefaultAdmin) => {
```

Comment:
Can we have this as a class method of `MemberInvitation`? Calling `invitation.sendEmail(...)` sounds right, it will also simplify the function and make it consistent with the rest of the business logic that mostly lives there (e.g. `accept`, `decline`)

Reply:
@Betree that's an excellent idea and I've implemented it in the latest commit. One caveat: `invitation.sendEmail` leads to an error, presumably because it's a circular reference (`sendEmail` hasn't been defined before exporting the class). So in my implementation I use `this.sendEmail`, which passes the new end-to-end test I've added in opencollective/opencollective-frontend#7710.

I've convinced myself that this is functionally equivalent since all the same parameters are passed, but of course I'd appreciate a reviewer's feedback.